src/components/kvitteringsside/Kvitteringsside.tsx

```diff
@@ -51,7 +52,7 @@ export default function Kvitteringsside({ personalia, innsendingstidspunkt }: Kv
             </p>
 
             <h4 className={styles.centerText}>
-                Takk for søknaden, {personalia.fornavn} {personalia.etternavn}!
+                Takk for søknaden, {personalia?.fornavn} {personalia?.etternavn}!
```

Comment:
Når er navn null? Skal man i det hele tatt kunne søke uten navn?

Reply:
? operatoren her sørger for at Kvitteringssteget ikke feiler med en klientsidefeil dersom personalia er `undefined`. Men det bør ikke kunne skje, strengt tatt... Kanskje bedre å bare droppe`?`?